airqo-platform/AirQo-api
# Mobile app api authentication exemptions

**_WHAT DOES THIS PR DO?_**
- Adds api authentication at the load balancer

**_WHAT ISSUES ARE RELATED TO THIS PR?_**
- Jira cards
    - [AN-160](https://airqoteam.atlassian.net/browse/AN-160?atlOrigin=eyJpIjoiNjZlMTViMzM4YzRmNDc1Zjk3OTQwNTMzZmYwMmU2NjIiLCJwIjoiaiJ9)

**_HOW DO I TEST OUT THIS PR?_**
 - Apply the staging ingess file
 - Attempt to get the devices 
 - You should get an "Unauthorized" response body
    ![image](https://user-images.githubusercontent.com/41787587/187031184-7f5ed228-c95b-4ae6-80c5-60f0a8e6dd3d.png)
 - Repeat the process but this time with an authentication token in the Authorization header
 - You should get the list of devices.
   ![image](https://user-images.githubusercontent.com/41787587/187031229-2e8a1a72-5293-489d-ae6c-0ba6c190dd06.png)

**_WHICH ENDPOINTS SHOULD BE READY FOR TESTING?:_** 
- Any endpoint

**_IS THERE ANY JENKINS CONSOLE LINK FOR CODE COVERAGE AND BUILD INFO?_**
- n/a

**_ARE THERE ANY RELATED PRs?_**
- #1131
